src/service/poc_lora.proto

```diff
@@ -160,6 +164,9 @@ message lora_invalid_beacon_report_v1 {
   /// The asserted elevation of the gateway in AGL ( above ground level)
   /// derived from gateway metadata
   int32 elevation = 6;
+  // provides any additional context for invalid reason
+  // for example the deny list version used as part of the deny list check
+  invalid_details invalid_details = 7;
```

Comment:
rename invalid_details to `details`.. i.e. `invalid_details details = 7;` to match the invalid_reason setup

Reply:
kept it as is, the reason field has two naming conventions depending on msg type, it will be one of "reason" or "invalid_reason"....ideally they would have been consistent but alas here we are
Rather than go with two naming conventions for the details field, i am using `invalid_details` naming in all types. This works better in the oracle side, allowing for a consistent naming convention to be utilised